docs/index.md

```diff
@@ -0,0 +1,329 @@
+# lolex
+
```

Comment:
Here is my stab at the first few sections.  I have limited experience with markdown and JavaScript (I have much more with Java), so please excuse any errors!
*******************************************************
# lolex

## About

**lolex** is a JavaScript package within Sinon.JS that enables you to implement timing related web APIs. For example, use **lolex** to simulate a system clock in your testing environment to control the flow of time when your testing scenario requires scheduling semantics.

Implement the following methods with **lolex**:
##COMMENT##
I left this list in, but I don't think this list should be in this section unless it is comprehensive. Also, I think you could/should take it out if the methods are listed in a navigation pane on the left as suggested in one of the threads)

* [`setTimeout`](#settimeoutcallback-timeout-args)
* [`clearTimeout`](#cleartimeoutid)
* [`setImmediate`](#setimmediatecallback-args)
* [`clearImmediate`](#clearimmediateid)
* [`setInterval`](#setintervalcallback-timeout-args)
* [`clearInterval`](#clearintervalid)
* [`requestAnimationFrame`](#requestanimationframecallback)
* [`cancelAnimationFrame`](#cancelanimationframeid)
* [`Date`](#date)
* [`nextTick`](#nexttickcallback-args)
* [`queueMicrotask`](#queuemicrotaskcallback)
* [`performance.now`](#performancenow)

## Installation

To install **lolex**, enter the following command in the terminal:

```shell
npm install lolex --save-dev
```
Alternately, you can [build it yourself](https://github.com/sinonjs/lolex/blob/master/package.json#L22)
##COMMENT## build **what** yourself?  A bundle?  Install package?  Please excuse my ignorance!)

Reply:
@rjseymour66 
Thank you very much for your feedback.I would like to adopt most of it, just some comments:
> ... enables you to implement timing related web APIs

I think _implement_ is a bit wrong here, because **lolex** already implements/overrides this methods. I would maybe write something like: _... enables you to use timing related web APIs_?

Same here:
> Implement the following methods with **lolex**:

_**lolex** implemnts the following methods:_

> I left this list in, but I don't think this list should be in this section unless it is comprehensive. Also, I think you could/should take it out if the methods are listed in a navigation pane on the left as suggested in one of the threads)

Yes, @benjamingr also suggested it as far as I remember. I think we can do this, there seem to be some extensions available to configure the Table of Content (https://python-markdown.github.io/extensions/toc/). I will have a look if this works for us. It seems a combination of `baselevel` and `toc_depth` should do the trick here.

> build **what** yourself? A bundle? Install package? Please excuse my ignorance!)

It's simply a package :grinning:

Reply:
Yeah that all sounds good - I think implements was a bit awkward, but I thought maybe it was an important word since it was used so much, haha.  "Use" is better!

I'll make the changes - thanks for the feedback!

Reply:
No worries, mate. I'm currently working on this. Let me quickly change this :smile:

Reply:
I applied the changes.
Preview: https://mgred.github.io/lolex/

Reply:
Just applied the following to the `mkdocs.yml`
```yml
markdown_extensions:
    - toc:
        toc_depth: 3
```
Against my assumption this does not add the 3rd level headings to the navigation on the left. Instead it enables 3rd level headings to be rendered in the Table of Content, a block which is identified as `[TOC]` in the markdown file and renders as:

![toc](https://user-images.githubusercontent.com/2325260/47754017-c51e3b80-dc99-11e8-9679-b347beeab02b.png)